### PR TITLE
Update to 0.36.0

### PR DESCRIPTION
**Pull request by aprasolova**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -53,6 +53,63 @@ neofs_ir__pprof_allow: []
...
+      proto_tick_interval: 2s
+      listen: "{{ neofs_ir__blockchain_p2p_addresses }}"
+      peers:
+        min: 1
+        max: 5
+        attempts: 20
+      ping:
+        interval: 30s
+        timeout: 90s
+      native_activations:
```
Comment (roman-khimov) [inline defaults/main.yml:101]: It's not a part of `p2p` and it can be safely omitted.

File: defaults/main.yml [hunk 2]
```diff
@@ -146,9 +195,9 @@ neofs_ir__config: []
 neofs_ir__group_config: []
 neofs_ir__host_config: []
 
-neofs_ir__combined_config: '{{ 
+neofs_ir__combined_config: '{{
   neofs_ir__default_config +
-  neofs_ir__config + 
+  neofs_ir__config +
```
Comment (roman-khimov) [inline defaults/main.yml:200]: Whitespace change -> separate commit.

File: defaults/main.yml [hunk 3]
```diff
@@ -53,6 +53,63 @@ neofs_ir__pprof_allow: []
...
+    max_traceable_blocks: 2102400
+    seed_nodes: "{{ neofs_ir__blockchain_seed_nodes }}"
+    rpc:
+      listen: "{{ neofs_ir__blockchain_rpc_addresses }}"
+    p2p:
+      dial_timeout: 5s
+      proto_tick_interval: 2s
+      listen: "{{ neofs_ir__blockchain_p2p_addresses }}"
+      peers:
+        min: 1
```
Comment (roman-khimov) [inline defaults/main.yml:95]: Where does it come from?

File: defaults/main.yml [hunk 4]
```diff
@@ -53,6 +53,63 @@ neofs_ir__pprof_allow: []
...
+        max: 5
```
Comment (roman-khimov) [inline defaults/main.yml:96]: And this one?

File: defaults/main.yml (hunk 3 above)
Comment (aprasolova) [reply defaults/main.yml:95]: From the sample config, both. What values should be there instead?

File: defaults/main.yml (hunk 2 above)
Comment (aprasolova) [reply defaults/main.yml:200]: I cannot do it due to the vim settings. It automatically cuts off whitespaces at the end of a line.

File: defaults/main.yml (hunk 3 above)
Comment (roman-khimov) [reply defaults/main.yml:95]: Node's `config.yaml` is just a sample and what you have here in `main.yml` is something that looks like a mainnet/testnet config. So I'd expect some values suitable for testnet/mainnet, and they're the same as in NeoGo configs for these networks: https://github.com/nspcc-dev/neo-go/blob/c50ab951645a1c901315ac5366170868245c34fd/config/protocol.mainnet.yml#L62

File: defaults/main.yml (hunk 2 above)
Comment (roman-khimov) [reply defaults/main.yml:189]: You can. You really really can. Irrespective of `$EDITOR` settings there are at least seven ways to deal with this problem. And should be solved.

**Comment by roman-khimov** [comment]
DCO, btw?

File: defaults/main.yml (hunk 2 above)
Comment (aprasolova) [reply defaults/main.yml:189]: Done

File: defaults/main.yml (hunk 3 above)
Comment (aprasolova) [reply defaults/main.yml]: Updated

**Comment by aprasolova** [comment]
> DCO, btw?

Fixed

File: defaults/main.yml [hunk 5]
```diff
@@ -190,4 +228,3 @@ neofs_ir__ferm__dependent_rules:
     protocol: 'tcp'
     role: 'neofs-ir'
     rule_state: "{{ 'present' if neofs_ir__control_enabled else 'absent' }}"
-
```
Comment (roman-khimov) [inline defaults/main.yml:193]: It's a whitespace change too.

File: defaults/main.yml [hunk 6]
```diff
@@ -53,6 +53,52 @@ neofs_ir__pprof_allow: []
...
+    max_traceable_blocks: 2102400
+    seed_nodes: "{{ neofs_ir__blockchain_seed_nodes }}"
+    rpc:
+      listen: "{{ neofs_ir__blockchain_rpc_addresses }}"
+    p2p:
+      dial_timeout: 3s
+      proto_tick_interval: 2s
+      listen: "{{ neofs_ir__blockchain_p2p_addresses }}"
+      peers:
+        min: 10
```
Comment (roman-khimov) [inline defaults/main.yml:95]: Let's set it to 4. It'd be better for 7-node networks with CNs only.

File: defaults/main.yml [hunk 7]
```diff
@@ -146,10 +184,10 @@ neofs_ir__config: []
 neofs_ir__group_config: []
 neofs_ir__host_config: []
 
-neofs_ir__combined_config: '{{ 
+neofs_ir__combined_config: '{{
```
Comment (roman-khimov) [inline defaults/main.yml:187]: All whitespace changes belong to a single commit.

File: defaults/main.yml (hunk 5 above)
Comment (aprasolova) [reply defaults/main.yml]: Done

File: defaults/main.yml (hunk 7 above)
Comment (aprasolova) [reply defaults/main.yml:187]: done

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**